storage/innobase/log/log0files_io.cc

```diff
@@ -1167,6 +1167,10 @@ static dberr_t log_check_file(const Log_files_context &ctx, Log_file_id file_id,
     ib::error(ER_IB_MSG_LOG_FILE_IS_EMPTY, file_path.c_str());
     return DB_ERROR;
   }
+
+  /* PXB logs redo log doesn't care about server redo log size
```

Comment:
PXB redo logs do not care about server redo log size restrictions and PXB has a single redo file and it can be > 4G size

Reply:
fixed